src/highdicom/content.py

```diff
@@ -1478,133 +1525,603 @@ def __init__(
+            self.ModalityLUTSequence = [modality_lut]
+        else:
+            if rescale_intercept is None:
+                raise TypeError(
+                    'Argument "rescale_intercept" must be specified when '
+                    '"modality_lut" is not specified.'
+                )
+            if rescale_slope is None:
+                raise TypeError(
+                    'Argument "rescale_slope" must be specified when '
+                    '"modality_lut" is not specified.'
+                )
+            if rescale_type is None:
+                raise TypeError(
+                    'Argument "rescale_type" must be specified when '
+                    '"modality_lut" is not specified.'
+                )
+            self.RescaleIntercept = format_number_as_ds(rescale_intercept)
+            self.RescaleSlope = format_number_as_ds(rescale_slope)
+            if isinstance(rescale_type, RescaleTypeValues):
+                self.RescaleType = rescale_type.value
+            else:
+                _check_long_string(rescale_type)
+                self.RescaleType = rescale_type
+
+
+class PresentationLUT(LUT):
+
+    """Dataset describing an item of the Presentation LUT Sequence."""
+
+    def __init__(
+        self,
+        first_mapped_value: int,
+        lut_data: np.ndarray,
+        lut_explanation: Optional[str] = None
+    ):
+        """
+
+        Parameters
+        ----------
+        first_mapped_value: int
+            Pixel value that will be mapped to the first value in the
+            lookup-table.
+        lut_data: numpy.ndarray
+            Lookup table data. Must be of type uint8 or uint16.
+        lut_explanation: Union[str, None], optional
+            Free-form text explanation of the meaning of the LUT.
+
+        """
+        super().__init__(
+            first_mapped_value=first_mapped_value,
+            lut_data=lut_data,
+            lut_explanation=lut_explanation
+        )
+
+
+class PresentationLUTTransformation(Dataset):
+
+    """Dataset describing the Presentation LUT Transformation as part of the
+    Pixel Transformation Sequence to transform polarity pixel values into
+    device-indendent presentation values (P-Values).
+
+    """
+
+    def __init__(
+        self,
+        presentation_lut_shape: Optional[
+            Union[PresentationLUTShapeValues, str]
+        ] = None,
+        presentation_luts: Optional[Sequence[PresentationLUT]] = None,
+    ):
+        """
+
+        Parameters
+        ----------
+        presentation_lut_shape: Union[highdicom.pr.PresentationLUTShapeValues, str, None], optional
+            Shape of the presentation LUT, applied after the Softcopy
+            VOI LUT Transformation to create display values.
+        presentation_luts: Optional[Sequence[highdicom.LUT]], optional
+            LUTs for the presentation LUT, applied after the Softcopy
+            VOI LUT Transformation to create display values.
+
+        """  # noqa: E501
```

Comment:
```suggestion
            VOI LUT Transformation to create display values.
            
        Note
        -----
        Only one of ``presentation_lut_shape`` or ``presentation_luts`` should be provided.

        """  # noqa: E501
```

May need to change this if the above parameter name changes

Reply:
Addressed via fa147db72fd6598b2d9ee8e1f4e97cdbceff1362